src/measure_theory/measure/measure_space.lean

```diff
@@ -966,6 +966,16 @@ assume t ht,
 calc μ.restrict s t = μ (t ∩ s) : restrict_apply ht
 ... ≤ μ t : measure_mono $ inter_subset_left t s
 
+lemma restrict_congr_set {m0 : measurable_space α} ⦃s s' : set α⦄ ⦃μ : measure α⦄
+  (hs : s =ᵐ[μ] s') : μ.restrict s = μ.restrict s' :=
+le_antisymm (restrict_mono' hs.le $ le_refl μ) (restrict_mono' hs.symm.le $ le_refl μ)
```

Comment:
This already exists as `measure_theory.restrict_congr_set`. Can you move that lemma, and `restrict_mono_ae` to a more appropriate place?